akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -84,14 +85,12 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
       logic ⇒
 
       final case class SubStream(
-        streamId:                  Int,
-        state:                     StreamState,
-        outlet:                    Option[BufferedOutlet[ByteString]],
-        inlet:                     Option[SubSinkInlet[ByteString]],
-        initialOutboundWindowLeft: Long
-      ) {
-        var outboundWindowLeft = initialOutboundWindowLeft
```

Comment:
Aha, good catch. So, all WINDOW_UPDATEs were lost if they were received before the response came in from the user (because that's when `copy` is used).